src/CMakeLists.txt

```diff
@@ -127,7 +127,7 @@ else()
 endif()
 
 target_include_directories("${PROGRAM_PREFIX}doom"
-                           PRIVATE "${CMAKE_CURRENT_BINARY_DIR}/../")
+                           PRIVATE "${CMAKE_CURRENT_BINARY_DIR}/../" "${PROJECT_SOURCE_DIR}/win32/")
```

Comment:
This appears to add the include path for all builds, including non-MSVC and non-win32, which looks wrong.

Reply:
That's a good call. Oops.